### PR TITLE
Add Nancy netcore sample

### PR DESCRIPTION
**Pull request by mikedevbo**

### CODE REVIEW

File: samples/web/send-from-nancy-module/Core_7/WebApplication.Core/Startup.cs [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+namespace WebApplication.Core
+{
+    public class Startup
+    {
+        public void Configure(IApplicationBuilder app, IApplicationLifetime applicationLifetime)
+        {
+			// Configure endpoint
+			var endpointConfiguration = new EndpointConfiguration("Samples.Nancy.Sender");
+			var transport = endpointConfiguration.UseTransport<LearningTransport>();
+			endpointConfiguration.UsePersistence<LearningPersistence>();
```
Comment (mauroservienti) [inline samples/web/send-from-nancy-module/Core_7/WebApplication.Core/Startup.cs]: Given that it’s a SendOnly endpoint there is need to define any persistence.

**Comment by mauroservienti** [comment]
@andreasohlund this sample is `.NET Core` only. Do we want to have a full framework version as well as we do for all the others, if supported?

I'm fine with sticking with a `.NET Core` only version.

**Comment by andreasohlund** [comment]
There is already a full FX version https://docs.particular.net/samples/web/send-from-nancy-module/

**Comment by andreasohlund** [comment]
Thank you so much @mikedevbo 👍

**Pull request merged**